docs/horizen_eon/06-service_providers/eon_service_providers.md

```diff
@@ -0,0 +1,76 @@
+---
+title: EON-compatible Service Providers
+---
+
+The EON ecosystem is perpetually changing with new partners and integrations. To accommodate the compatibility of EON platform with third-party integration, several service provider protocols are used to facilitate the on-boarding of new dApps, services, tools, and more.
+
+See the [EON Ecosystem](https://eon.horizen.io/ecosystem) for a description of current service providers.    
+
+
+# Service Providers: Technology Features
+
+As new partners and integrations are on-boarding to the EON ecosystem, the need to use the following service providers’ technologies is imperative for a successful EVM-compatible sidechain:
+
+
+## Ankr  
+
+Ankr provides a Web3 infrastructure that has many use cases, from Web3 API, staking, blockchain deployment tool, and smart contract execution. 
+
+Request for [EON Ankr-specific](http://info@horizen.io) documentation. 
```

Comment:
I think we are trying to make this an email link. If so, it should be like the following:

```suggestion
Request for [EON Ankr-specific](mailto:info@horizen.io) documentation. 
```